documentation/synthetic_tokens/tutorials/liquidation.md

```diff
@@ -6,11 +6,15 @@ The prompt and accurate execution of liquidations and disputes is a core assumpt
 Liquidation and dispute bots, as described below and implemented [here](https://github.com/UMAprotocol/protocol/tree/master/liquidator) and [here](https://github.com/UMAprotocol/protocol/tree/master/disputer), are infrastructure tools that will help maintain the overall health of the UMA ecosystem.
 They are currently compatible with the priceless synthetic token contract template, as described [here](../explainer.md) and implemented [here](https://github.com/UMAprotocol/protocol/tree/master/core/contracts/financial-templates).
 
+### Liquidation vs Dispute Bot
+
 The liquidation bot monitors all open positions within a given expiring multi-party contract and liquidates positions if their collateralization ratio, as inferred from off-chain information about the value of the price identifier, drops below a given threshold.
 
 The dispute bot monitors all liquidations occurring within a given expiring multi-party contract and initiates disputes against liquidations it deems invalid, as inferred from off-chain information about the value of the price identifier.
 A liquidation is invalid if a position was correctly collateralized at the time of liquidation.
 
+In short, a liquidation bot is to liquidate under-collateralized positions while a dispute bot is used for disputing inaccuracies within the UMA DVM system.
```

Comment:
Do we mention anywhere that only a dispute triggers a price request? (or, if the position is not liquidated and is allowed to expire normally)

Reply:
I did not know this, do you think I should squeeze this in? Or you think this is fine and we could mention that elsewhere?

Reply:
I think its worth mentioning in a short sentence.